days-since-curiosity-on-mars.js

```diff
@@ -0,0 +1,12 @@
+const now = new Date();
+const currentTimestamp = now.getTime();
+const daysSinceCuriosityLanded = () => {
+  const then = new Date("2012-08-06");
+  const landingTimestamp = then.getTime();
+  const millisecondsInADay = 1000 * 60 * 60 * 24;
+  const days = Math.floor(
+    (currentTimestamp - landingTimestamp) / millisecondsInADay
+  );
+  return `${days}`;
```

Comment:
Here you are unnecessarily converting a number to a string. You could just return the number instead:
```suggestion
  return days;
```